### PR TITLE
Bcc Members Client

### PR DESCRIPTION
**Pull request by yuriy-butkevych**

### CODE REVIEW

File: BccMembers.Api.Client/IBccMembersApiClient.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System.Threading.Tasks;
+using BccMembers.Api.Client.Contracts;
+
+namespace BccMembers.Api.Client
+{
+    public interface IBccMembersApiClient
+    {
+        Task<BccPerson> GetPersonAsync(int id);
```
Comment (tkristiansen) [inline BccMembers.Api.Client/IBccMembersApiClient.cs]: I think we should use personId as paramenter name to be clear

File: BccMembers.Api.Client2/BccMembersApiClient.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Net.Http;
```
Comment (tkristiansen) [inline BccMembers.Api.Client2/BccMembersApiClient.cs]: Seems like there are a couple of redundant files here in Client2.

Comment (yuriy-butkevych) [reply BccMembers.Api.Client2/BccMembersApiClient.cs]: Do you mean usings @tkristiansen? All usings are currently used

File: BccMembers.Api.Client/IBccMembersApiClient.cs (hunk 1 above)
Comment (yuriy-butkevych) [reply BccMembers.Api.Client/IBccMembersApiClient.cs]: 👍

File: BccMembers.Api.Client2/BccMembersApiClient.cs (hunk 2 above)
Comment (yuriy-butkevych) [reply BccMembers.Api.Client2/BccMembersApiClient.cs]: right, I see

**Pull request merged**